tests/conftest.py

```diff
@@ -644,6 +645,25 @@ async def _setup_mqtt_entry():
         yield _setup_mqtt_entry
 
 
+@pytest.fixture(autouse=True)
```

Comment:
This was causing random failures because it wasn't mocked out and would sometimes win the race to log about not being able to determine the ip

Reply:
Apparently it was also slowing down the CI